packages/docs/stories/avatar.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import README from '@availity/avatar/README.md';
+import Avatar from '@availity/avatar';
+
+storiesOf('Components|Avatar', module)
+  .addParameters({
+    readme: {
+      // Show readme at the addons panel
+      sidebar: README,
+      // eslint-disable-next-line react/prop-types
+      StoryPreview: ({ children }) => <div>{children}</div>,
+    },
+  })
+  .add('default', () => (
+    <div className="py-3">
+      <Avatar />
```

Comment:
Since the images aren't going to load when this is deployed we should add a fallback text knob and set the default to something like [placeholder image](https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png)

And then also adding a default width just for the example so the loader doesn't span the whole width of the screen

Reply:
997309a421cba24ea8434ef238a26c2f3499c109